backend/infrahub/core/manager.py

```diff
@@ -603,7 +606,25 @@ async def get_one_by_hfid(
         if not node_schema.human_friendly_id or len(node_schema.human_friendly_id) != len(hfid):
             raise NodeNotFoundError(branch_name=branch.name, node_type=kind, identifier=hfid_str)
 
-        filters = {node_schema.human_friendly_id[idx]: item for idx, item in enumerate(hfid)}
+        # TODO: create a function like `hfid_to_graphql_filters` for this?
```

Comment:
reminder to remove the TODO before merging
I don't know if this logic will be needed elsewhere, but it seems like it fits within this method for now